doc/1.manual/1.introduction.txt

```diff
@@ -13,7 +13,7 @@ or online:
     http://msp.ucsd.edu/Pd_documentation/
 
 More extensive FLOSS documentation is available on:
-    http://en.flossmanuals.net/PureData (English)
+    https://en.flossmanuals.net/pure-data/_full/ (English)
     http://fr.flossmanuals.net/PureData (Francais)
```

Comment:
that should be https:// as well.

(and really: the floss-manuals are horribly outdated)

Reply:
> that should be https:// as well.

You mean the French one?

> (and really: the floss-manuals are horribly outdated)

I agree. We pull them off here?

Reply:
How about:

```
More extensive documentation is available on:
    http://puredata.info/docs/BooksAboutPd/
```